tests/FSharpPlusFable.Tests/FSharpTests/General/Monad.fs

```diff
@@ -18,31 +23,58 @@ let monad = testList "Monad" [
         equal ["Join"] (SideEffects.get ()))
     #endif
 
-    #if !FABLE_COMPILER
+    #if !FABLE_COMPILER || FABLE_COMPILER_3
     testCase "workFlow" (fun () ->       
         let testVal = 
             monad {
                 let! x1 = WrappedListD [1;2]
                 let! x2 = WrappedListD [10;20]
                 return ((+) x1 x2) }
-        Assert.IsInstanceOf<WrappedListD<int>> (testVal))
-
+        #if !FABLE_COMPILER
+        Assert.IsInstanceOf<WrappedListD<int>> (testVal)
+        #endif
+        equal (WrappedListD [11; 21; 12; 22]) testVal
```

Comment:
Isn't this a stronger requirement than the one above?

Reply:
I just enable the type test and it works, thanks for pointing out.